custom/icds_reports/migrations/sql_templates/create_functions.sql

```diff
@@ -27,8 +27,10 @@ CREATE OR REPLACE FUNCTION update_location_table() RETURNS VOID AS
 $BODY$
 DECLARE
 	_ucr_location_table text;
+  _ucr_aww_tablename text;
```

Comment:
nit: looks like this is mixing tabs and spaces - alignment is off in the github UI

Reply:
Haha, oh, just saw the next commit.